gazebo/rendering/Visual.cc

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  *
 */
+#include <math.h>
```

Comment:
I can not find why the include is needed in the changes, does it come from previous changes where the C isnan was used?

Reply:
fixed ;)